src/www/app/app.ts

```diff
@@ -153,9 +154,11 @@ export class App {
 
   showLocalizedError(e?: Error, toastDuration = 10000) {
     let messageKey: string;
-    let messageParams: string[] | undefined;
-    let buttonKey: string;
-    let buttonHandler: () => void;
+    let messageParams: string[] = [];
+    let buttonKey = 'error-details';
```

Comment:
We should only set the `buttonKey` if we actually have details to show.

For the errors listed below, I don't think it's useful to show the details button, unless the Error has a message or a cause. The error type/name is already captured by the message.

Here is my proposal:
If error is explicitly handled, show the details only if we have a message or a cause.
For unknown errors, always show the error details.

I realized that the stack is not super useful and a bit distracting. Perhaps we can output something like:
```js
`${e.toString()}${e.cause ? `\nCause: ${e.cause.toString()}` : ''}`
```
The `Error.toString` method seems to output `${name}: ${message}`, without the stack: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/toString#description